doc/interface_cli/cli_datasets.rst

```diff
@@ -0,0 +1,89 @@
+.. _cli_datasets:
+
+CLI datasets
+===============
+
+To install a dataset (from the example datasets or your custom dataset) use the command:
+
+.. prompt:: bash
+
+    peerannot install installationFile.py
+
+For either case, a Python file describing the installation will be needed.
+
+Example datasets
+----------------
+
+To install an example dataset, only the installation file is needed. For example
+want to install the ``cifar10H`` library, run:
+
+.. prompt:: bash
+
+    cd datasets/cifar10H & peerannot install cifar10h.py
+
+
+Custom datasets
+---------------
+
+To install your dataset, you will have to use the ``customDataset.py`` installation file located at `datasets/`
+and add multiple arguments depending on the structure of your dataset.
+
+Only the ``answers`` file and the ``answers-format`` arguments must always be included, for more information about
+the answers formats see the format page:
+
+Taskless dataset
+^^^^^^^^^^^^^^^^
+
+If your dataset has no task, then you can add the ``no-task`` flag with the ``answers`` and ``answers-format`` argument.
+
+.. prompt:: bash
+
+    cd datasets/MyDataset & peerannot install ../customDataset.py --no-task --answers answersFile.json --answers-format 1
+
+Dataset with tasks
+^^^^^^^^^^^^^^^^^^
+
+In the case your dataset has tasks (if you want to train a model for image classification). 
+A ``train-set`` must be included and you will have to specify its path.
+A ``files-path`` also has to be given. It should include the path to the file with the same
+order as the one in the ``answers`` file.
+
+A validation set can be provided with the ``val-set`` option but is not mandatory. In case
+a validation set is not provided it will be created with 20% of the train set.
+
+Finally, label names can be provided in a file with the option ``label-names`` which can help
+construct the structure of the dataset (especially if the test set has no ground truth file).
+In case it's not given, it will be assumed that the structure of the dataset is similar to a Pytorch
+ImageFolder dataset (see https://pytorch.org/vision/main/generated/torchvision.datasets.ImageFolder.html)
+where tasks are arranged inside folders per labels.
+
+Here are some examples of commands to create custom datasets:
```

Comment:
Bonne idée ! On peut en faire une sous-section (pour le référencement par tag sur la page ensuite) ? Maybe dataset installation examples (more than creation, it's an install)

Reply:
Et pareil, si on peut éviter le scroll